match/lib/match/encrypt.rb

```diff
@@ -104,6 +103,12 @@ def crypt(path: nil, password: nil, encrypt: true)
         success = false
       end
 
+      # to show an error message if something goes wrong
```

Comment:
you meant this inside the previous `unless`?

Reply:
no, that is the old comment of the `dev/null` stuff that is now the code below this line - it makes sure if the `openssl` command fails with an error, that this error message is output to the command line of the user if `verbose` is set. Should I rewrite this somehow?

Reply:
but you are printing this when on verbose and regardless if there was an error.

Reply:
Oh. My brain...

Reply:
So: Yes, this was meant to go into the `unless`. Moved it there and also moved the comments around to match.